client/my-sites/upgrades/map-domain/index.jsx

```diff
@@ -1,136 +1,146 @@
 /**
  * External dependencies
  */
-var page = require( 'page' ),
-	React = require( 'react' ),
-	omit = require( 'lodash/omit' ),
-	{ connect } = require( 'react-redux' );
+import page from 'page';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-var HeaderCake = require( 'components/header-cake' ),
-	MapDomainStep = require( 'components/domains/map-domain-step' ),
-	{ DOMAINS_WITH_PLANS_ONLY } = require( 'state/current-user/constants' ),
-	cartItems = require( 'lib/cart-values' ).cartItems,
-	upgradesActions = require( 'lib/upgrades/actions' ),
-	observe = require( 'lib/mixins/data-observe' ),
-	wpcom = require( 'lib/wp' ).undocumented(),
-	paths = require( 'my-sites/upgrades/paths' );
-import { currentUserHasFlag } from 'state/current-user/selectors';
+import HeaderCake from 'components/header-cake';
+import MapDomainStep from 'components/domains/map-domain-step';
+import { DOMAINS_WITH_PLANS_ONLY } from 'state/current-user/constants';
+import { cartItems } from 'lib/cart-values';
+import upgradesActions from 'lib/upgrades/actions';
+import wp from 'lib/wp';
+import paths from 'my-sites/upgrades/paths';
 import Notice from 'components/notice';
+import { currentUserHasFlag } from 'state/current-user/selectors';
+import isSiteUpgradeable from 'state/selectors/is-site-upgradeable';
```

Comment:
As far as I remember idea here is to import it as follows:
```js
import { isSiteUpgradeable } from 'state/selectors';
```

For Webpack those 2 might be different chunks of code ... Not sure though :)